mbuild/path.py

```diff
@@ -95,6 +119,37 @@ def _path_history(self):
         """
         pass
 
+    def visualize(self, bead_color="#5d8aa8"):
```

Comment:
We should add a bead size parameter here, since the radii and bond lengths of these paths are often at coarse-grained length scales, the default bead size of 0.3 can look pretty small. Maybe add a parameter and use a default of 0.7.